### ISSUE TITLE
Exceptions on 4xx and 5xx response status codes

### ISSUE THREAD

**Issue by PercyDwn** [clicksports/lexoffice-php-api#121]
When I try to request something and the lexoffice api answers with a 4xx or 5xx status code, a GuzzleException and after that a LexOfficeApiException is thrown.

This is not exactly the kind of behaviour I expect, as I want to be able to display the error which I received. Therefore I need the response and its body.

Currently I have a workaround: I simply changed the method for sending the request in `\Clicksports\LexOffice\App:getResponse` on Line 144 from

`$this->client->send()` to  `$this->client->sendRequest()`.

But I'm still interested why it's currently handled like that?

EDIT:
I just found the closed issue #120 , where @Sysix already wrote how to access the response.
> You can access the response body with $lexOfficeException->getPrevious()->getResponse()->getBody()->__toString();

So basically this issue is redundant. Maybe this information should be added to the readme?

**Comment by Sysix** [comment]
> This is not exactly the kind of behaviour I expect.

There is only one reason why I wrote it like this:
When you have mutiple guzzle calls from outside, you can still handle the lexoffices one separately.

> $this->client->send() to $this->client->sendRequest().
> 
> But I'm still interested why it's currently handled like that?

Maybe in the future we will accept other HTTP Client which are compatible with [PSR-18](https://www.php-fig.org/psr/psr-18/).
But there is this issue: https://github.com/guzzle/guzzle/issues/3088 so it will problably a major update ;) 

> Maybe this information should be added to the readme?

#122
